tests/test_main.py

```diff
@@ -0,0 +1,141 @@
+"""Run from the `pyta` root directory to use the local `python_ta` rather than
+installed `python_ta` package.
+"""
+
+import subprocess
+import sys
+from os import environ, path
+
+import python_ta
+
+# Find the source root directory
+source = path.dirname(__file__)
+while path.basename(source) != "pyta":
+    source = path.dirname(source)
+
+# Gives absolute path to source root directory (pyta)
+SOURCE_ROOT = source
+# SOURCE_ROOT = path.normpath(path.join(path.dirname(__file__), ".."))  # NOTE: Professor may prefer this approach
```

Comment:
This is indeed the approach I want you to use. You can also import the correct relative path from `python_ta.config` (see my above comment).

In general, you should not have top-level code other than defining constants (so while loops aren't good). You also shouldn't assume the root-level folder is called `pyta`, it could be called something else.